### DISCUSSION TITLE
Visualize Model Outputs Button Not Working on GUI

### DISCUSSION THREAD

**Discussion by penalemesc** [talmolab/sleap#1915] [Help!]
Hi! I was trying to use the Visualize Model Outputs button (on the GUI) to be able to see the confidence maps of my models but I have not been able to get it to show anything. I have tried multiple models, one with the final tag even, but no matter what I do the training_config.json, intial_config.json and even an export of the entire model as a compressed .json file do not display anything. I don't know if I am doing something wrong and maybe that is why I cannot see anything, so if I could get any guidance that would be appreciated.

Used 3 bottom-up models for testing
Software versions:
SLEAP: 1.3.3
TensorFlow: 2.7.0
Numpy: 1.21.6
Python: 3.7.12
OS: Windows-10-10.0.22621-SP0

(Installed through the first method of the documentation)

**Comment by roomrys** [comment]
Hi @penalemesc,

This is something that has not been working, but did not seem to have priority (no squeaky wheels, no attention). There is a closed pull request on this:
- #1104

that got a little bloated with feature creep (and thus unfinished before needing to move on to another task). 

But, if it is a feature that people value and want to use, then we can definitely prioritize it and get it fixed up - you are the second person both in this week (and in a year) to ask about it, so it is now on the radar.

Sorry for your troubles, but thank you for letting us know that you would like to use the Visualize Model Outputs feature and you can expect to see it in our future release.

Thanks,
Liezl

**Reply by penalemesc** [reply]
Hi @roomrys ! 

Oh! Okay, awesome, good to know. Is there anything you could recommend me doing to see where and why the classifier might not be working fully well yet? In a sort of plot or graph visual?

Thank you in advance

**Reply by talmo** [reply]
Hi @penalemesc,

Are you still having issues getting a good model working?

If so, do you mind sharing some screenshots and the `training_job_config.json` for your bottom up models?

Cheers,

Talmo
